src/v2/cookbook/packaging-sfc-for-npm.md

```diff
@@ -57,11 +57,11 @@ src/
 dist/
 ```
 
-<p class="tip">Throughout this document, references are made to the package.json file listed above. The file used in these examples was generated by hand, and will include the minimum configuration required for the discussion/task at hand. It is likely your own package.json file will contain a lot more than is listed here.</p>
+<p class="tip">A través de este documento, las referencias están echas al package.json listado arriba. El archivo usado en estos ejemplos fue generado a mano, e incluirá la configuración mínima requerida par la discusión/tarea del momento. Es posible que su propio package.json contendrá mucho más de lo que se expone acá</p>
 
-### How does npm know which version to serve to a browser/build process?
+### ¿Cómo sabe npm cuál versión servir al navegador/proceso compilado?
```

Comment:
would be good to change to something like _¿Cómo sabe npm cuál versión servir al navegador o proceso de compilación?_ are you agree?